hacspec-derive/src/lib.rs

```diff
@@ -0,0 +1,407 @@
+extern crate hacspec;
```

Comment:
No need for any `extern crate` in Rust 2018 :slightly_smiling_face:

Reply:
It's a limitation of Rust that crate that define `derive` proc macros cannot export anything else, hence the `hacspec-derive` crate... And in those special "derive" crates you need to declare `extern` crates for some reason :)

Reply:
ah, well proc macros are weird...